2019-05-20-tidb-weekly.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Weekly update (May 13 ~ May 19, 2019)
+date: 2019-05-20
+summary: Last week, we landed 37 PRs in the TiDB repository, 19 PRs in the TiSpark repository, and 24 PRs in the TiKV and PD repositories.
+tags: ['TiDB', 'TiSpark', 'TiKV', 'PD']
+---
+
+# Weekly update in TiDB
+
+Last week, we landed [37 PRs](https://github.com/pingcap/tidb/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Amerged+merged%3A2019-05-13..2019-05-19+) in the TiDB repository.
+
+## Added
+
+- [Implement `ALTER DATABASE` to alter the charset/collation](https://github.com/pingcap/tidb/pull/10393)
+- [Support the `incremental` and `repeats` importer comments](https://github.com/pingcap/tidb/pull/10434)
```

Comment:
s/comments/columns/

Reply:
@zz-jason I think it should be "columns" instead. Will update it. Thanks.

Reply:
Fixed. https://github.com/pingcap/weekly/pull/98